common/config/rush/.pnpmfile.cjs

```diff
@@ -1,3 +1,10 @@
+function overrideToWorkspace(pkg, dependencies) {
+  for (const dep of dependencies) {
+    pkg.dependencies[dep] = "workspace:*";
+    pkg.peerDependencies[dep] = "workspace:*";
```

Comment:
do you need to set the peer dep? i guess it doesnt really matter since itll be installed anyway but just curious if this was needed / used to just suppress peer warnings further

Reply:
Yeah this was mainly to avoid all those unmet peerDependency warnings, which are just noise